lib/src/util/validation_util.dart

```diff
@@ -101,13 +103,13 @@ class ValidationUtil {
   ///
   /// Useful for verifying warnings in unit tests.
   ///
-  /// > See: [startRecordingValidationWarnings]
+  /// > See: `startRecordingValidationWarnings`
   static ValidationUtilWarningCallback onWarning;
 }
 
 class ValidationWarning extends Error {
   /// Message describing the problem.
-  final message;
+  String message;
```

Comment:
This should remain final
```suggestion
  final String message;
```